components/sysutils/smp_utils/manifests/sample-manifest.p5m

```diff
@@ -52,37 +81,66 @@ file path=usr/bin/smp_zone_lock
 file path=usr/bin/smp_zone_unlock
 file path=usr/bin/smp_zoned_broadcast
 file path=usr/include/scsi/smp_lib.h
-file path=usr/lib/libsmputils1.a
-link path=usr/lib/libsmputils1.so target=libsmputils1.so.1.0.0
-link path=usr/lib/libsmputils1.so.1 target=libsmputils1.so.1.0.0
-file path=usr/lib/libsmputils1.so.1.0.0
-file path=usr/share/man/man1m/smp_conf_general.1m
-file path=usr/share/man/man1m/smp_conf_phy_event.1m
-file path=usr/share/man/man1m/smp_conf_route_info.1m
-file path=usr/share/man/man1m/smp_conf_zone_man_pass.1m
-file path=usr/share/man/man1m/smp_conf_zone_perm_tbl.1m
-file path=usr/share/man/man1m/smp_conf_zone_phy_info.1m
-file path=usr/share/man/man1m/smp_discover.1m
-file path=usr/share/man/man1m/smp_discover_list.1m
-file path=usr/share/man/man1m/smp_ena_dis_zoning.1m
-file path=usr/share/man/man1m/smp_phy_control.1m
-file path=usr/share/man/man1m/smp_phy_test.1m
-file path=usr/share/man/man1m/smp_read_gpio.1m
-file path=usr/share/man/man1m/smp_rep_broadcast.1m
-file path=usr/share/man/man1m/smp_rep_exp_route_tbl.1m
-file path=usr/share/man/man1m/smp_rep_general.1m
-file path=usr/share/man/man1m/smp_rep_manufacturer.1m
-file path=usr/share/man/man1m/smp_rep_phy_err_log.1m
-file path=usr/share/man/man1m/smp_rep_phy_event.1m
-file path=usr/share/man/man1m/smp_rep_phy_event_list.1m
-file path=usr/share/man/man1m/smp_rep_phy_sata.1m
-file path=usr/share/man/man1m/smp_rep_route_info.1m
-file path=usr/share/man/man1m/smp_rep_self_conf_stat.1m
-file path=usr/share/man/man1m/smp_rep_zone_man_pass.1m
-file path=usr/share/man/man1m/smp_rep_zone_perm_tbl.1m
-file path=usr/share/man/man1m/smp_utils.1m
-file path=usr/share/man/man1m/smp_write_gpio.1m
-file path=usr/share/man/man1m/smp_zone_activate.1m
-file path=usr/share/man/man1m/smp_zone_lock.1m
-file path=usr/share/man/man1m/smp_zone_unlock.1m
-file path=usr/share/man/man1m/smp_zoned_broadcast.1m
+file path=usr/lib/$(MACH64)/libsmputils1.a
+link path=usr/lib/$(MACH64)/libsmputils1.so target=libsmputils1.so.1.0.0
+link path=usr/lib/$(MACH64)/libsmputils1.so.1 target=libsmputils1.so.1.0.0
+file path=usr/lib/$(MACH64)/libsmputils1.so.1.0.0
+file path=usr/sbin/$(MACH32)/smp_conf_general
+file path=usr/sbin/$(MACH32)/smp_conf_phy_event
```

Comment:
$(MACH32) doesn't look correct if we have BUILD_BITS= 64

Reply:
Yes, I know.
I tried to use the options for configure but to no avail.
The installation procedure copies the same files to several directories.
The files in usr/bin are 64bit binaries and they are used in smp_utils.p5m.